src/scmdata/netcdf.py

```diff
@@ -293,10 +293,10 @@ def run_to_nc(df, fname, dimensions=("region",), extras=()):
         ds.created_at = datetime.utcnow().isoformat()
         ds._scmdata_version = __version__
 
-        if hasattr(df, "metadata"):
-            ds.setncatts(df.metadata)
+        if hasattr(run, "metadata"):
```

Comment:
how does dealing with metadata work on reading in? I assume it's lost?

Reply:
Nope all the metadata are preserved (the newly read obj also now includes _scmdata_version and created_at fields). An example is in the netcdf notebook

Reply:
Also should be covered by test_netcdf.py::test_nc_with_metadata

Reply:
Beauty